### ISSUE TITLE
trying to display events from two different tables in mysql how?

### ISSUE THREAD

**Issue by vincentdabs** [Serhioromano/bootstrap-calendar#756]

**Comment by vincentdabs** [comment]
<?php
include_once("db_connect.php");
$sqlEvents = "SELECT * FROM evs_event";
$resultset = mysqli_query($conn, $sqlEvents) or die("database error:". mysqli_error($conn));
$calendar = array();
$event = array();

while( $rows = mysqli_fetch_assoc($resultset) ) {	
	// convert  date to milliseconds
	$start = strtotime($rows['event_startDate']) * 1000;
	$end = strtotime($rows['event_endDate']) * 1000;	
	$calendar[] = array(
        'id' =>$rows['event_id'],
        'title' => $rows['event_desc'],
		'class' => 'event-success',
        'start' => "$start",
        'end' => "$end"
    );
    $event[] = array(
        'id' =>'293',
        'title' => 'This is warning class event with very long title to check how it fits to evet in day view',
		'class' => 'event-success',
        'start' => "1362938400000",
        'end' => "1363197686300"
    
    );
}
$calendarData = array(
	"success" => 1,	
    "result"=>$calendar,$event);
echo json_encode($calendarData);
exit;
?>

**Comment by R-Caser** [comment]
$sqlEvents = "SELECT * FROM **table1** WHERE xy ='yx'";
    $resultset = mysqli_query($conn, $sqlEvents) or die("database error:".mysqli_error($conn));
    $calendar = array();

    while( $rows = mysqli_fetch_assoc($resultset) ) {
        // convert date to milliseconds
        $calendar[] = array(
        'id' =>$rows['id'],
        'title' =>$dt." ". $rows['title'],
        'url' => "#",
        "class" => 'event-important',
        'start' => $rows["start"],
        'end' => $rows["end"]
        );
    }

$sqlEvents = "SELECT * FROM **table2** WHERE xy ='yx'";
    $resultset = mysqli_query($conn, $sqlEvents) or die("database error:".mysqli_error($conn));
    $calendar = array();

    while( $rows = mysqli_fetch_assoc($resultset) ) {
        // convert date to milliseconds
        $calendar[] = array(
        'id' =>$rows['id'],
        'title' =>$dt." ". $rows['title'],
        'url' => "#",
        "class" => 'event-important',
        'start' => $rows["start"],
        'end' => $rows["end"]
        );
    }

**Comment by vincentdabs** [comment]
can i also ask why the calendar is not displaying in chrome and mozilla in other computers? but in my own its displaying?

**Comment by cablegunmaster** [comment]
Yes , because you don't host it online . Only on your own pc? (assumption)
Buy some webstorage and webadress and put all your files there.  

Either that or you are hosting at your own pc but never is connected from outside. which are port issues (port forwarding required).
